core-bundle/tests/Fragment/Reference/ContentElementReferenceTest.php

```diff
@@ -20,7 +20,7 @@ class ContentElementReferenceTest extends TestCase
 {
     public function testCreatesTheControllerNameFromTheModelType(): void
     {
-        $model = new ContentModel();
+        $model = (new \ReflectionClass(ContentModel::class))->newInstanceWithoutConstructor();
```

Comment:
`new ContentModel()` executes the constructor of the `DcaExtractor` which loads the default language file. So this test previously had a hidden dependency on a test that preloads the language file.

Reply:
You should use `mockClassWithProperties` for this

Reply:
fixed in bf16286cbe1a894bc06be4f2e1e56f9aa7d4b701